script/api.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+import psycopg2
+from flask import Flask, jsonify, request, session
+
+app = Flask(__name__)
+
+hostname = os.getenv('POSTGRES_HOST')
+database = os.getenv('POSTGRES_DB')
+username = os.getenv('POSTGRES_USER')
+
+port_id = 5432
+conn = psycopg2.connect(
+    host=hostname,
+    dbname=database,
+    user=username,
+    port=port_id
+)
+
+cur = conn.cursor()
+
+
+@app.route('/login', methods=['POST'])
+def login():
+
+    email = request.json.get('email')
+    password = request.json.get('password')
+    cur.execute("SELECT * FROM users WHERE email='{}' ".format(email))
+    user_with_email = cur.fetchone()
+    
+    if user_with_email :
+        cur.execute(
+            "SELECT * FROM users WHERE email='{}' AND password=crypt('{}',password)".format(email, password))
+        user = cur.fetchone()
+        if user:
+            id = user[0]
```

Comment:
Can we do it like this 
`id, username= user[0], user[1]`
rather than using user[1] across the code